src/Components/TodoContainer.js

```diff
@@ -1,74 +1,97 @@
+
+        try {
+            const response = await fetch(url, options);
+
+            if (!response.ok) {
+                console.log("Airtable error response:");
+                console.log(response);
+                throw new Error(`Error: ${response.status}`);
+            }
+
+            // Parse the response JSON
+            const data = await response.json();
+            console.log('API Response Data:', data);
+
+            // Transform API data.records into todo objects
+            const todos = data.records.map((record) => ({
+                id: record.id,
+                title: record.fields.title,
+                createdTime: new Date(record.createdTime),
+            }));
+
+            // Update todoList and isLoading states
+            setTodoList(todos);
+            setIsLoading(false);
+        } catch (error) {
+            console.log(`Fetch error: ${error.message}`);
+        }
+    }, [isAscending]);
+
+    // Fetch data on initial render and whenever sorting order changes
+    useEffect(() => {
+        fetchData();
+        // eslint-disable-next-line 
+    }, [fetchData]);
+
+    // Function to toggle sorting order and sort the todoList accordingly
+    function toggleSortingOrder() {
+        // Toggle between ascending and descending order
+        setIsAscending(!isAscending);
+
+        // Sort the todoList based on the new sorting order
+        setTodoList((prevTodoList) =>
+            [...prevTodoList].sort((a, b) => {
+                return isAscending ? a.createdTime < b.createdTime ? -1 : 1 : a.createdTime > b.createdTime ? -1 : 1;
+            })
+        );
+    }
+
+
+
+    // Define the removeTodo function to remove a todo item
+    function removeTodo(id) {
+        // Filter the todoList array to exclude the todo item with the specified id
+        const updatedTodoList = todoList.filter((todo) => todo.id !== id);
+
+        // Update the todoList state with the new array of todos
+        setTodoList(updatedTodoList);
     }
-  }
-
-  // Fetch data on initial render
-  useEffect(() => {
-    fetchData();
-  }, []);
-
-  // Define the removeTodo function to remove a todo item
-  function removeTodo(id) {
-    // Filter the todoList array to exclude the todo item with the specified id
-    const updatedTodoList = todoList.filter((todo) => todo.id !== id);
-
-    // Update the todoList state with the new array of todos
-    setTodoList(updatedTodoList);
-  }
-
-  return (
-    <>
-      <TodoList
-        todoList={todoList}
-        onRemoveTodo={removeTodo}
-        isLoading={isLoading}
-      />
-    </>
-  );
+
+    return (
+        <>
+            <button
+                onClick={toggleSortingOrder}
+                className={styles.ToggleButton}
+            >
+                Toggle Sorting Order: {isAscending ? "Ascending" : "Descending"}
```

Comment:
I think we want this button to display the sort you'd be switching to if you click it, not the current sort? 
```suggestion
                Toggle Sorting Order: {isAscending ? "Descending" : "Ascending"}
```